CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/docker-infrastructure/compare/release-0.3.3...master
+### Fixed
+- Bump log4j dependency to 2.17.2
```

Comment:
Do we want to reference issue key, so that in the future someone who will review this would know the context/reason behind the upgrade?

Reply:
added